content/en/agent/autodiscovery/template_variables.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Autodiscovery Template Variables
+kind: documentation
+disable_toc: true
+further_reading:
+- link: "/agent/autodiscovery/integrations"
+  tag: "Documentation"
+  text: "Create and load an Autodiscovery Integration Template"
+- link: "/agent/autodiscovery/ad_identifiers"
+  tag: "Documentation"
+  text: "Match a container with the corresponding Integration Template"
+- link: "/agent/autodiscovery/management"
+  tag: "Documentation"
+  text: "Manage which Container to include in the Agent Autodiscovery"
+---
+
+Use the following Template Variables when configuring Autodiscovery in order to dynamically assign your containers values:
```

Comment:
Not all variables are compatible with all autodiscovery listeners, we should probably have a table like this https://github.com/DataDog/datadog-agent/tree/master/pkg/autodiscovery/listeners#template-variable-support